### DISCUSSION TITLE
Gallery plug-in

### DISCUSSION THREAD

**Discussion by ritchiemarkov** [bootscore/bootscore#533] [Ideas]
Hello guys, 

it's not replated to the theme its self but why dont we create a CPT Gallery plugin?

Something like modula.

I have started to develope one, and its pretty functioning (but a lot of messy gpt code). 

Back-end photo upload / sorting works well. Shortcode works well also, Displaying pretty ok. 

Why im posting it here?

We can unite various swiper plugins into one, and use templates for generating galleries / elements (so we have 1 swiper file point).

So idk, we can actually add some various templates as masonry also. 

What do you think about?

**Comment by crftwrk** [comment]
Hi Richard,

well, it's a nice idea, but I do not think this fits to this repo. Because each plugin here creates a lot of text like docs, readme etc., must be updated from time to time and somebody must answer all those issues and discussions which a new plugin will create. Our goal is to reduce plugins, making existing ones better to reduce maintaining work. I'm not sure if a gallery CPT plugin is it worth to do all this other stuff and this should be better done by the community.

There was a similar case a few months ago. A developer wanted to add some more templates to the theme that fits to his project but not for everyone. He created now an own repo for this https://github.com/kreincke/bsToppings and everyone who wants to use them can simply download files from his repo.

What we can do is to create a new repo which contains a list with all plugins/themes/tools created by users, pin it to the top and create a menu-link to it on bootscore.me. Developers can simply add their project to the list by a pull request. So this should walk by itself.

Some projects to start:

- https://github.com/kreincke/bsToppings
- https://github.com/crftwrk/bs-swiper-gallery
- Gallery CPT
- ...

Agree?

**Reply by ritchiemarkov** [reply]
Hello Basti! 

Please can you check my plugin?

I'm pretty sattisfied and I know there is a big margin for improvement.

I will upload it ASAP to github.


I want to implement your layouts for cards etc inside. So I can use one single plugin for all.

But the results amazes me since I made it alone. (chatgpt ofc did it)

**Reply by ritchiemarkov** [reply]
https://drive.google.com/file/d/1A8FcifADxQRaFJAsa7yuiSWptz0f8jwE/view?usp=drivesdk

**Reply by crftwrk** [reply]
Please publish on GitHub ;-)

**Comment by ritchiemarkov** [comment]
Can you take a look? I will upload in 5 mins.

Il Mar 1 Ago 2023, 17:55 Basti ***@***.***> ha scritto:
